server/lib/models/channel.go

```diff
@@ -43,6 +56,18 @@ func NewChannel(dbif Dbif, a *Account, req *proto.ChannelCreateRequest) (*Channe
 	return ch, nil
 }
 
+func FindChannel(dbif Dbif, id proto.UUID) (*Channel, error) {
+	c := &Channel{
+		proto.Model_Channel {
+			Id: id,
+		},
+	}
+	if err := dbif.SelectOne(c, dbm.Stmt("select * from %s.channels where id=$1"), c.Id); err != nil {
```

Comment:
__[gometalinter]__
* [gotype] invalid operation: c (variable of type *Channel) has no field or method Id